src/frontend/Komponenter/Behandling/Totrinnskontroll/SendTilBeslutterFooter.tsx

```diff
@@ -29,6 +31,15 @@ const HovedKnapp = styled(Button)`
     margin-right: 1rem;
 `;
 
+export enum FremleggsoppgaveType {
+    INNTEKT = 'INNTEKT',
+}
+
+export interface IFremleggsOppgave {
+    kanOppretteFremleggsoppgave: boolean;
+    fremleggsoppgaveTyper: FremleggsoppgaveType[];
```

Comment:
```suggestion
    oppgavetyper: FremleggsoppgaveType[];
```
Ville endret navn på feltet til oppgavetyper. Feltet ligger under IFremleggsOppgave så tenker det blir redundant slik det er nå

Reply:
Det kan komme flere typer av fremleggsoppgaver, selv om vi bare har en type for inntekt nå